### PR TITLE
Copy editing: uncertainty

### PR DESCRIPTION
**Pull request by lglattly**
### Description

This pull request is to copy edit and mark examples in the uncertainty subpackage.

### CODE REVIEW

File: docs/uncertainty/index.rst [hunk 1]
```diff
@@ -166,14 +171,18 @@ Additionally, Poisson and uniform |distribution| creation functions exist::
...
 Users are free to create their own distribution classes following similar
 patterns.
 
-
 Using Distributions
 -------------------
 
-This object now acts much like a |quantity| or |ndarray| for all but the
+.. EXAMPLE START: Using and Accessing Distributions
+.. not quite sure what to call this example or how best to break it up
```
Comment (mhvk) [inline docs/uncertainty/index.rst:183]: Maybe `[Accessing] Properties of Distributions`? I think I would not break it up further.

File: docs/uncertainty/index.rst [hunk 2]
```diff
@@ -237,16 +250,20 @@ discussion of covariances)::
...
-One of the main applications for distributions is unceratinty propagation, which
+One of the main applications for distributions is uncertainty propagation, which
 critically requires proper treatment of covariance. This comes naturally in the
-Monte Carlo sampling approach used by the |distribution| class, as long as
+Monte Carlo sampling approach used by the |Distribution| class, as long as
 proper care is taken with sampling error.
 
-To start with a simple example, two un-correlated distributions should produce
+.. EXAMPLE START: Covariance in Distributions
+.. also not quite sure how to break up this chunk of examples
```
Comment (mhvk) [inline docs/uncertainty/index.rst:264]: Is it needed? It does seem a logical set.

File: docs/uncertainty/index.rst (hunk 1 above)
Comment (lglattly) [reply docs/uncertainty/index.rst:183]: Good to know and yes that title is better!

**Comment by bsipocz** [comment]
Double approval, so I go ahead and push the button. Thank you @lglattly!

**Pull request merged**